packages/http/src/mocker/index.ts

```diff
@@ -89,12 +89,24 @@ function negotiateResponse(
   input: IPrismInput<IHttpRequest>,
   resource: IHttpOperation,
 ) {
-  if (input.validations.length > 0) {
+  const {
+    [DiagnosticSeverity.Error]: errors,
+    [DiagnosticSeverity.Warning]: warnings,
+  } = groupBy(input.validations, (validation) => validation.severity);
+
+  if (errors) {
     return handleInputValidation(input, resource);
   } else {
     return pipe(
-      withLogger(logger =>
-        logger.success({ name: 'VALIDATOR' }, 'The request passed the validation rules. Looking for the best response'),
+      withLogger(logger => {
+          if (warnings) {
+            warnings.forEach((warn) => {
+              logger.warn({ name: 'VALIDATOR' }, warn.message);
+            });
```

Comment:
You can make this a single line

`warnings.forEach(warn => logger.warn({ name: 'VALIDATOR' }, warn.message))`